app/models/expression_file_info.rb

```diff
@@ -0,0 +1,42 @@
+class ExpressionFileInfo
+  include Mongoid::Document
+
+  embedded_in :study_file
+
+  field :library_construction_protocol, type: String
+  field :units, type: String
+  field :biosample_input_type, type: String
+  field :multimodality, type: String
+  field :is_raw_counts, type: Boolean, default: false
+
+  # note that species and reference genome/annotation live at the study_file level, not here
+
+  UNITS_VALUES = [nil, 'UMI-corrected raw counts', 'raw counts']
+  validates :units, inclusion: {in: UNITS_VALUES}
+
+  BIOSAMPLE_INPUT_TYPE_VALUES = [nil, 'Whole cell', 'single nuclei', 'bulk']
+  validates :biosample_input_type, inclusion: {in: BIOSAMPLE_INPUT_TYPE_VALUES}
+
+  MULTIMODALITY_VALUES = [nil, 'CITE-seq', 'Patch-seq']
+  validates :multimodality, inclusion: {in: MULTIMODALITY_VALUES}
+
+  LIBRARY_CONSTRUCTION_VALUES = [nil,
+                                 'Smart-seq2/Fluidigm C1',
+                                 'Mars-seq',
```

Comment:
Per https://science.sciencemag.org/content/343/6172/776 and https://www.nature.com/articles/s41596-019-0164-4:
```suggestion
                                 'MARS-seq',
```

And similarly for other references.

(Either MARS-seq or MARS-Seq -- the first part is always all-capitalized.)